### PR TITLE
added the ability to change file name

### PR DESCRIPTION
**Pull request by Marounissa**

### CODE REVIEW

File: vast-perf.sh [hunk 1]
```diff
@@ -380,100 +381,102 @@ EOF
...
-      exit 1
-  esac
-  shift
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --name=*) file_name="${1#*=}" ;;
+        --vms=*)
+            mVIP="${1#*=}"
+            ;;
+        --proxy=*)
```
Comment (breuner) [inline vast-perf.sh]: the indentation of several lines in the parameter check section got changed unintentionally.

**Review by breuner — COMMENTED** [review]
hi @Marounissa, generally all good with additing this. only small problem is the indentation change that i mentioned. can you please resubmit without that?

File: vast-perf.sh [hunk 2]
```diff
@@ -90,6 +90,7 @@ ADMINUSER="admin"
 ADMINPASSWORD=123456
 LOOPBACK=1 # only applies when running on cnodes. default is on now. BUT: this requires a lot of vips..
 VERBOSE=1 # verbosely output details, may be too noisy on large clusters
+file_name="randrw"
```
Comment (cpeel) [inline vast-perf.sh]: This ends up being the prefix for the filename, but it's actually the fio job name. I'd prefer to see it be called `JOB_NAME` so users aren't surprised when this isn't actually the name of the file that gets created.

File: vast-perf.sh [hunk 3]
```diff
@@ -380,100 +381,102 @@ EOF
...
-      HELPME="true"
-      ;;
-    *)
-      showusage
-      exit 1
-  esac
-  shift
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --name=*) file_name="${1#*=}" ;;
```
Comment (cpeel) [inline vast-perf.sh]: `--job-name` would be a better name for this argument, placed near `--jobs` below. It also needs to be added to the usage statement.

**Review by cpeel — CHANGES_REQUESTED** [review]

**Review by cpeel — APPROVED** [review]
I addressed the PR comments and tested this on selab-var-201.

**Pull request merged**